resources/google-music-electron-old.svg

```diff
@@ -0,0 +1,488 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Not sure it is. This is an older attempt at an icon that I made, and I don't believe it was in the repo before now. I had mentioned concerns about using an identical headphones icon, which is likely copyrighted (even though I redrew it, it's pretty much 1:1 with the original logo)

A better name might have been "alt".